### DISCUSSION TITLE
Is there a way to support both webmock and fakeweb?

### DISCUSSION THREAD

**Discussion by zben** [vcr/vcr#857] [Ideas]
**Is your feature request related to a problem? Please describe.**
I have a project that uses different client libraries. Some needs webmock and some needs fakeweb. If I use just one, some requests will fail to be intercepted. 

**Describe the solution you'd like**
Can we have a way to specify adapter on a per test case level to override the global config?

**Describe alternatives you've considered**
Not sure

**Additional context**
No.

**Comment by krainboltgreene** [comment]
While I'm for it, it would require a bit of work to achieve. Feel free to submit a PR.
